pyinstaller-gui.py

```diff
@@ -76,7 +76,11 @@ def killapp(self, event):
         sys.exit(0)
 
     def makePackage(self, event):
-        commands = ['python3', 'pyinstaller.py']
+        if sys.platform.startswith('win'):
+            pythoncmd = ['py', '-3']
```

Comment:
https://docs.python.org/3/using/windows.html
3.4.1.1. From the command-line
It is only implemented in 3.3+ .

So if 3.0-3.2 have to be supported, another way to acces python3 is needed.
Maybe directly from the registry?